test/common/util.js

```diff
@@ -83,6 +87,16 @@ describe('common/util', function() {
     assert.deepEqual(REQUEST_DEFAULT_CONF, { pool: { maxSockets: Infinity } });
   });
 
+  it('should export an error for module instantiation errors', function() {
+    var missingProjectIdError = new Error([
```

Comment:
Is this really useful? Won't it mean that we have to change the text twice if we want to modify it?